.github/workflows/CI.yml

```diff
@@ -0,0 +1,135 @@
+name: CI
+# Run on master, tags, or any pull request
+on:
+  pull_request:
+    branches: "*"
+  push:
+    branches: master
+    tags: "*"
+  schedule:
+    - cron: "0 2 * * *"  # Daily at 2 AM UTC (8 PM CST)
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        version:
+          - "1.0"  # LTS
+          - "1"    # Latest release
+          - nightly
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+          - x86
+        exclude:
+          # Test 32-bit only on Linux
+          - os: macOS-latest
+            arch: x86
+          - os: windows-latest
+            arch: x86
+    steps:
```

Comment:
Do we want Slack notifications for this project in #nightly-dev?

Reply:
We could. Typically I just handle failures on this repo which I'd notice by using CCMenu. Looks like I'll need to find an alternative.

Reply:
Oo CCMenu is neat! Yeah not too sure if you want to open that up since you're the defacto maintainer of this package.

Reply:
I'm not sure what I want to do at the moment. I'm going to punt on this as I don't want to hold up: https://github.com/JuliaTime/TimeZones.jl/pull/315

Reply:
Fair enough, can always come back to it in the future!